test/spec/hierarchical-facets/unknown-facet.js

```diff
@@ -20,10 +18,5 @@ test('hierarchical facets: throw on unknown facet', function(t) {
     }]
   });
 
-  t.throws(
-    bind(helper.toggleRefine, helper, 'unknownFacet', 'beers'),
-    'Refine on an unknown hierarchical facet throws'
-  );
-
-  t.end();
+  expect(bind(helper.toggleRefine, helper, 'unknownFacet', 'beers')).toThrow();
```

Comment:
Can we use an inline snapshot now that we use Jest?

Reply:
Nope because we don't have [Prettier](https://jestjs.io/docs/en/snapshot-testing#inline-snapshots) installed. Once we setup the stack for it we can.